var/spack/repos/builtin/packages/touchdetector/package.py

```diff
@@ -47,7 +47,8 @@ class Touchdetector(CMakePackage):
     depends_on('eigen', when='@4.5:')
     depends_on('fmt', when='@4.5:')
     depends_on('hdf5@1.8:')
-    depends_on('morphio@2.0.8:', when='@4.5:')
+    depends_on('morphio@2.0.8:', when='@4.5:5.1')
```

Comment:
Isn't the morhpio dependency transitive from morpho-kit ?

Reply:
It's not a direct dependency, though. Should be pulled in through `morpho-kit`.